build.xml

```diff
@@ -611,6 +637,36 @@ ${git_status}
     <exec command="git reset --hard" passthru="true" />
   </target>
 
+  <!-- Reset database and configuration of the demo setup -->
+  <target name="reset_setup" description="reset configuration and database of demo">
+    <property name="solr_pid_file" value="${localdir}/solr-8983.pid" />
+    <!-- fail if the system is already running -->
+    <if>
+      <not><available file="${solr_pid_file}" /></not>
```

Comment:
This logic has been reversed. Do the comment and message need to be changed to match?

Reply:
Yes, fixed.